server/health.go

```diff
@@ -42,3 +59,16 @@ func (s *Server) checkCRIHealth(ctx context.Context, timeout time.Duration) erro
 
 	return nil
 }
+
+func (s *Server) cniPluginReadinessCheck(ctx context.Context) {
+	cniInitOnce.Do(func() {
+		go func() {
```

Comment:
Would this work avoid the additional `go func`?
```suggestion
	go cniInitOnce.Do(func() {
```

Reply:
The idea is to start the readiness check immediately but run the actual plugin check asynchronously.